src/sagemaker/xgboost/model.py

```diff
@@ -108,16 +108,15 @@ def __init__(
         self.model_server_workers = model_server_workers
 
     def prepare_container_def(self, instance_type, accelerator_type=None):
-        """Return a container definition with framework configuration set in model environment
-            variables.
+        """Return a container definition with framework configuration set in model environment variables.
```

Comment:
from the failed build:

```
[7;33m************* Module sagemaker.xgboost.model[0m
src/sagemaker/xgboost/model.py:111:0: C0301: [1mLine too long (105/100)[0m ([1mline-too-long[0m)
```

can you split this line up? (and the second line should be indented at the same level as `"""`)